src/communication_pkg/scripts/HUD.py

```diff
@@ -61,8 +61,10 @@ def speed_callback(data):
     Stores averaged values in info
     """
     global info, speed_bufl, speed_bufr, speed_idx
-    speed_bufl[speed_idx] = data.data[0]
-    speed_bufr[speed_idx] = data.data[1]
+
+    speed_bufl[speed_idx] = data.data
+    speed_bufr[speed_idx] = data.data
```

Comment:
May want to remove left/right distinction in future.

Reply:
I moved the speed message to only have one value. This script is the only one that would have required a significant rewrite to utilize the new single valued message.